src/main/java/org/apache/rocketmq/dashboard/service/impl/OpsServiceImpl.java

```diff
@@ -49,6 +55,8 @@
     @Override
     public void updateNameSvrAddrList(String nameSvrAddrList) {
         configure.setNamesrvAddr(nameSvrAddrList);
+        // when update namesrvAddr, clean the mqAdminExt objects pool.
+        mqAdminExtPool.clear();
```

Comment:
Good for clear the pool~
Beside this, there is also LTS and VIPChannel config etc.